Cohere/Types/RerankRequest.cs

```diff
@@ -0,0 +1,45 @@
+
+namespace Cohere.Types;
+
+/// <summary>
+/// The object to be sent to the rerank endpoint
+/// </summary>
+public class RerankRequest: ICohereRequest
+{
+    // Visit https://docs.cohere.com/reference/rerank#request for more information on the input fields
+
+    /// <summary>
+    /// The model to use for text reranking
+    /// </summary>
+    public string? Model { get; set; } = "rerank-english-v3.0";
```

Comment:
this looks like a constant or config that should be doc or place somewhere else.